content/billing-pricing/pricing.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Pricing
+weight: 2
+---
+
```

Comment:
Should there be an intro about the free tier and what's included as well to explain the whole model?

Reply:
OK, I can see this is addressed later in the file.